index.html

```diff
@@ -0,0 +1,241 @@
+            <li class="css">css</li>
+            <li class="jvscrp">javascript</li>
+          </ul>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+      </div>
+      <!--card works four-->
+      <div class="card-Works-three">
+        <div class="Portfolio">
+          <img src="img/Portfolio3.png" alt="Portfolio pic" />
+
+          <h2 class="multi">Multi-Post Stories</h2>
+          <h3 class="CANOPY">CANOPY</h3>
+          <div class="Counter_one">
+            <img src="img/Counter1.png" alt="Counter1 pic" />
+          </div>
+          <h4 class="Back_text">Back End Dev</h4>
+          <div class="Counter_two">
+            <img src="img/Counter2.png" alt="Counter1 pic" />
+          </div>
+          <h5 class="date">2015</h5>
+          <p class="Primary-Text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="jvscrp">javascript</li>
+          </ul>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+      </div>
+    </section>
+    <!--About section-->
+    <section class="about">
+      <h1 class="abt-txt">About Myself</h1>
+      <p class="info">
+        Hello I’m a software developer! I can help you build a product
+        feature or a website, please look through some of my work and experience! If you
+        like what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+      <h2 class="Lets">LET’S CONNECT</h2>
+      <ul class="social_Media-two">
+        <li>
+          <a><img src="img/twitter.svg" alt="twitter_imge" /></a>
+        </li>
+        <li>
+          <a><img src="img/Linkedin.svg" alt="Linkedin_imge" /></a>
+        </li>
+        <li>
+          <a><img src="img/Vec.svg" alt="Media_imge" /></a>
+        </li>
+        <li>
+          <a><img src="img/Github.svg" alt="Github_imge" /></a>
+        </li>
+        <li>
+          <a><img src="img/Vector.svg" alt="vector_imge" /></a>
+        </li>
+      </ul>
+      <button type="submit" class="button">Get my resume</button>
+      <ul>
+        <li class="arrow"></li>
+        <li class="langa">Languages</li>
+      </ul>
+      <div class="fjva">
+        <img src="img/java1.png" alt="Javascript icone" />
+        <p>javaScript</p>
+      </div>
+      <div class="fhtml">
+        <img src="img/html.png" alt="html icone" />
+        <p>HTML</p>
+      </div>
+      <div class="fcss">
+        <img src="img/css.png" alt="css icone" />
+        <p>CSS</p>
+      </div>
+      
+      <ul>
+        <li class="arrow_one"></li>
+        <li class="Frame">Frameworks</li>
+        <li class="line_one"></li>
+        <li class="arrow_two"></li>
+        <li class="skll">Skills</li>
+        <li class="line_two"></li>
+      </ul>
+    </section>
```

Comment:
In this part of your project, You were asked to build only "Portfolio: setup and mobile version skeleton" that's why I won't be able to review the Work and About section for now. 👍🏻